packages/database/src/core/version.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** The semver (www.semver.org) version of the SDK. */
+export let SDK_VERSION = '';
```

Comment:
I believe this export in combination with `setSDKVersion`  doesn't work, as it exports the current state of the variable:

deps.js:

```
let foo = 'foo'
function setToBar() {
	foo = 'bar';
}
module.exports = {foo, setToBar};
```

```
let deps = require('./deps');
console.log(deps.foo);
deps.setToBar();
console.log(deps.foo);
```

Prints:

```
foo
foo
```

Reply:
It actually works because esm only exports references. The code compiled to cjs format would look like:
```javascript
exports.foo = 'foo';
function setToBar() {
    exports.foo = 'bar';
}
exports.setToBar = setToBar;
```